src/Razor/vs-threading.MainThreadAssertingMethods.txt

```diff
@@ -0,0 +1 @@
+[Microsoft.VisualStudio.Editor.Razor.JoinableTaskContextExtensions]::AssertUIThread
```

Comment:
❓What is this file?

Reply:
It adds additional methods to be considered by the vs-threading analyzers. This entry ensures that we won't get analyzer messages about VS COM objects needing to be accessed on the UI method if our `AssertUIThread` method is called. See [vs-threading.MainThreadAssertingMethods.txt](https://github.com/dotnet/razor/pull/9911/files/0ccb31a387cc13d6c81f39bbab655c0e7b1ca3f9#diff-111d193cc4e4217cb221c0e326683e5a4be6af1a2d9e178a1f7fda513d1a17d8) for more detail.